.gitignore

```diff
@@ -569,4 +569,5 @@ frontend/cypress/screenshots/*
 frontend/cypress/videos/*
 
 reports/
-frontend/reports/
\ No newline at end of file
+frontend/reports/
+legacy/src/main/resources/application-dev-mariamar.properties
```

Comment:
Don't add personal properties on classpaths, always use the config folder at the root of the application

Reply:
Sure! Thanks